modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/cmake-lists.mustache

```diff
@@ -27,28 +25,55 @@ endif()
 # BUILD TYPE
 message("A ${CMAKE_BUILD_TYPE} build configuration is detected")
 
-# Update require components as necessary
-#find_package(Boost 1.45.0 REQUIRED COMPONENTS ${Boost_THREAD_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${Boost_REGEX_LIBRARY} ${Boost_DATE_TIME_LIBRARY} ${Boost_PROGRAM_OPTIONS_LIBRARY} ${Boost_FILESYSTEM_LIBRARY})
+if (UNIX)
+
+message(STATUS "Building client library for Linux/Unix")
+
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -fPIC -Wall -Wno-unused-variable")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall -Wno-unused-variable")
+
+find_package(cpprestsdk REQUIRED)
+find_package(Boost REQUIRED)
 
-# build and set path to cpp rest sdk
-set(CPPREST_ROOT ${PROJECT_SOURCE_DIR}/3rdParty/cpprest)
-set(CPPREST_INCLUDE_DIR ${CPPREST_ROOT}/include)
-set(CPPREST_LIBRARY_DIR ${CPPREST_ROOT}/lib)
+else(UNIX)
```

Comment:
I'd use newer syntax - simply
`else()` - less confusing for users from other languages